greenplumpython/order.py

```diff
@@ -12,7 +10,9 @@
 
 class DataFrameOrdering:
     """
-    Represents an ordered GreenplumPython :class:`~dataframe.DataFrame` generated by
+    Represent an ordered GreenplumPython :class:`~dataframe.DataFrame`.
+
+    The order is decided by:
```

Comment:
![image](https://user-images.githubusercontent.com/449296/219598102-7d6959a5-392d-4aac-beea-93905401a940.png)

the generated doc seems confusing.